process/mock/storerMock.go

```diff
@@ -81,26 +63,23 @@ func (sm *StorerMock) Has(_ []byte) error {
 	return errors.New("not implemented")
 }
 
+// SearchFirst -
+func (sm *StorerMock) SearchFirst(key []byte) ([]byte, error) {
+	return sm.Get(key)
+}
+
 // RemoveFromCurrentEpoch -
 func (sm *StorerMock) RemoveFromCurrentEpoch(key []byte) error {
-	return sm.Remove(key)
+	delete(sm.data, string(key))
+	return nil
 }
 
 // Remove -
 func (sm *StorerMock) Remove(key []byte) error {
-	sm.mut.Lock()
```

Comment:
is this ok? This is not safe to be ran concurrently anymore

Reply:
Reverted (copy-pasted another mock by mistake).